.github/workflows/python2-3-app.yml

```diff
@@ -0,0 +1,43 @@
+name: DB12 testing
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 2.7]
+        
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      
+    - name: Add conda to system path
+      run: |
+        echo $CONDA/bin >> $GITHUB_PATH
+        
+    - name: Set up Conda and Install dependencies
+      run: |
+        conda create -c conda-forge -c free -n python python=${{ matrix.python-version }}
+        conda info --env
```

Comment:
Is it necessary?

Reply:
To set up the conda environment I had the option to either do it this way or to use an environment file so I just went with this one.

Reply:
Alright, for the first line `conda create ...` I understand, but what about the second one: `conda info --env`?

Reply:
Oh no that's not necessary I was just using it to debug and forgot to delete it